src/styles/ForecastSummaries.css

```diff
@@ -0,0 +1,35 @@
+.forecast-summaries {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.forecast-summary {
+  width: 200px;
+  height: 200px;
+  background-color: white;
+  border-radius: 10px;
+  margin: 20px;
+  text-align: center;
+}
```

Comment:
You have a conflict with this class, as there is another one with the same name in `ForecastSummary.css` . It should be removed from here and whatever properties you want should be set there. 
If you want to see how this is happening in the browser, inspect the ForecastSummary component and look at the styles block. Youll see that both instances of the class definitions are being rendered, and some are being overwritten by the others